xbmc/cores/AudioEngine/Sinks/AESinkDirectSound.cpp

```diff
@@ -119,56 +104,42 @@ std::unique_ptr<IAESink> CAESinkDirectSound::Create(std::string& device,
   return {};
 }
 
-bool CAESinkDirectSound::Initialize(AEAudioFormat &format, std::string &device)
+bool CAESinkDirectSound::Initialize(AEAudioFormat& format, std::string& device)
 {
   if (m_initialized)
     return false;
 
   bool deviceIsBluetooth = false;
-  LPGUID deviceGUID = nullptr;
-  RPC_WSTR wszUuid  = nullptr;
+  GUID deviceGUID = {};
   HRESULT hr = E_FAIL;
   std::string strDeviceGUID = device;
-  std::list<DSDevice> DSDeviceList;
-  std::string deviceFriendlyName;
-  DirectSoundEnumerate(DSEnumCallback, &DSDeviceList);
-
-  if(StringUtils::EndsWithNoCase(device, std::string("default")))
-    strDeviceGUID = GetDefaultDevice();
+  std::string deviceFriendlyName = "unknown";
```

Comment:
Not seems necessary initialize with "unknown" as this value never will be displayed (in normal circumstances).

Reply:
under normal circumstances. I thought it would be better than a blank in abnormal circumstances.